beacon-chain/db/validator.go

```diff
@@ -61,6 +61,31 @@ func (db *BeaconDB) ValidatorIndex(pubKey []byte) (uint64, error) {
 	return index, err
 }
 
+// ValidatorIndices accepts a slice of public keys and returns the corresponding map of validators indexes and public keys.
+func (db *BeaconDB) ValidatorIndices(pubKeys [][]byte) (map[uint64][]byte, error) {
+	m := make(map[uint64][]byte)
+	if !db.HasValidators(pubKeys) {
+		return m, fmt.Errorf("one or more of the validators\n%#x\ndoes not exist", pubKeys)
```

Comment:
Please don't use newlines in error message

Reply:
removed